Deadly skills

LIttleAncientForestKami opened the pull request:
mergin within same repo through github seems silly and overkill

LIttleAncientForestKami committed:
Skills updated

The more I C, the less I see.

LIttleAncientForestKami committed:
List
made out of skills

Measuring programming progress by lines of code is like measuring aircraft building progress by weight.
      -- Bill Gates

codeschool-kiddo commented on introduction.md:
```diff
@@ -4,3 +4,10 @@ My fave path for now seems to be Git one. :-)
 Favourite path: Git
```
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

LIttleAncientForestKami committed:
History kill as a skill
as per request

The real design action is in the memory sub-system - caches, buses, bandwidth and latency
      -- Richard Sites (DEC Alpha Architect)
